.travis.yml

```
@@ -0,0 +1,6 @@
+language: node_js
+node_js: "current"
```

```suggestion
node_js: "node"
```
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/
> The easiest way to specify Node.js versions is to use one or more of the latest releases in your .travis.yml:
> 
> **node latest stable Node.js release**
> iojs latest stable io.js release
> lts/* latest LTS Node.js release